backend/package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "server",
+  "name": "iscsc.fr-backend",
   "version": "0.1.0",
   "description": "Backend of the iscsc.fr website",
+  "private": true,
```

Comment:
simply not sure what `private` is supposed to do or enforce :yum:

Reply:
It simply forbid to publish package via `npm publish` https://docs.npmjs.com/cli/v6/configuring-npm/package-json#private

Reply:
makes sense!